kubectl_deploy/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: djangosecret
+type: Opaque
+data:
+  secretKey: dGhpc2lzVGhlU2VjcmV0S2V5Tm90aGluZ1RvU2VlSGVyZS4=
```

Comment:
No hardcoded secrets in the repo, please. Pass it in with an environment variable.

Reply:
It is being passed in with an environment variable. The deployment.yaml is calling this secrets file and passing the value as an env var. Normally you would encrypt this file but due to the fact this is a quick start, this would create too much complication. A developer would have to have their GPG key added to the repo to perform the quick start. If we make it explicit in the documentation to have this file encrypted in a production environment would this suffice?

Reply:
@tatyree that's obviously generally true, but this is a special case. Because this is a reference app that serves as a tutorial, we need our users to be able to check this repo out and see how it all slots together with a minimum of fuss, so we've decided to not encrypt secrets, and accept the near-zero risk of them being available in the clear. The risk is so low because this is just a demo app, and the secrets we have aren't credentials for anything 'real', like AWS or similar. If we do end up with AWS creds, then we will have to encrypt those, or only check in dummy values.